docs/content/how-to/node-tuning.md

```diff
@@ -0,0 +1,114 @@
+
+    Save the ConfigMap manifest in a file called `tuned-1.yaml`:
+    ```
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: tuned-1
+      namespace: clusters
+    data:
+      tuned: |
+        apiVersion: tuned.openshift.io/v1
+        kind: Tuned
+        metadata:
+          name: tuned-1
+          namespace: openshift-cluster-node-tuning-operator
+        spec:
+          profile:
+          - data: |
+              [main]
+              summary=Custom OpenShift profile
+              include=openshift-node
+
+              [sysctl]   # Section with minimum changes
+              vm.dirty_ratio="55"
+            name: tuned-1-profile
+          recommend:
+          - match:
+            - label: tuned-1-node-label
+            priority: 20
+            profile: tuned-1-profile
+    ```
+
+    Create the ConfigMap in the management cluster:
+    ```
+    oc --kubeconfig="$MGMT_KUBECONFIG" create -f tuned-1.yaml
+    ```
+
+    Reference the ConfigMap in the NodePools `spec.tunedConfig` field, either by editing an existing NodePool or creating a new NodePool.
+    ```
+    apiVersion: hypershift.openshift.io/v1alpha1
+    kind: NodePool
+    metadata:
+    ...
+    name: nodepool-1
+    namespace: clusters
+    ...
+    spec:
+    ...
+    tunedConfig:
+    - name: tuned-1
+    status:
+    ...
+    ```
+
+2. Now that the ConfigMap containg a Tuned manifest has been created and referenced in a NodePool, the Node Tuning Operator will sync the Tuned objects into the hosted cluster. You can check which Tuneds are defined, which profiles are set for each Node, and by labeling a Node with the label `tuned-1-node-label`, apply the custom profile we created to a Node.
+
+    List the Tuned objects in the hosted cluster:
+    ```
+    oc --kubeconfig="$HC_KUBECONFIG" get Tuneds -n openshift-cluster-node-tuning-operator
+    ```
+
+    Example output:
+    ```
+    NAME       AGE
+    default    7m36s
+    rendered   7m36s
+    tuned-1    65s
+    ```
+   
+    By default, NTO applies some default tunings to all worker Nodes. List the Profiles in the hosted cluster:
+
+    ```
+    oc --kubeconfig="$HC_KUBECONFIG" get Profiles -n openshift-cluster-node-tuning-operator
+    ```
+
+    Example output:
+    ```
+    NAME                           TUNED            APPLIED   DEGRADED   AGE
+    worker-1                       openshift-node   True      False      7m43s
+    worker-2                       openshift-node   True      False      7m14s
+    ```
+
+    As we can see, currently only the default Profiles are applied. If we label a Node in the hosted cluster, we should see that NTO applies the custom profile we defined earlier to that node.
+    ```
+    oc --kubeconfig="$HC_KUBECONFIG" label node/worker-1 tuned-1-node-label=""
```

Comment:
I'm not sure we should encourage users to use custom labels on the nodes like this.  When the NodePool has the default `.spec.management.upgradeType = Replace` upgrade type, they will be wiped on NodePool upgrade.  Another option is to add a warning about this and perhaps recommend the `InPlace` upgrade type?  Feedback/comments from the HyperShift team very welcome here.

nit: shell expands `""` as an empty string, so this can be completely omitted.

Reply:
Agreed, I removed the node label matching from the example Tuned CR, and added a node explaining this consideration underneath the CR. Now the example CR uses NodePool based matching,